it/src/test/java/org/corfudb/universe/scenario/FileDescriptorLeaksIT.java

```diff
@@ -0,0 +1,68 @@
+package org.corfudb.universe.scenario;
+
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.runtime.collections.CorfuTable;
+import org.corfudb.universe.GenericIntegrationTest;
+import org.corfudb.universe.UniverseManager.UniverseWorkflow;
+import org.corfudb.universe.group.cluster.CorfuCluster;
+import org.corfudb.universe.node.client.CorfuClient;
+import org.corfudb.universe.node.server.CorfuServer;
+import org.corfudb.universe.scenario.fixture.Fixture;
+import org.corfudb.universe.spec.FileDescriptorLeaksSpec;
+import org.corfudb.universe.universe.UniverseParams;
+import org.junit.Test;
+
+import static org.corfudb.universe.scenario.fixture.Fixtures.TestFixtureConst.DEFAULT_STREAM_NAME;
+import static org.junit.jupiter.api.Assertions.fail;
+
+@Slf4j
+public class FileDescriptorLeaksIT extends GenericIntegrationTest {
+
+    /**
+     * Test cluster behavior after one node down.
+     * <p>
```

Comment:
I think this test is too verbose to what it's trying to check. You can write a unit test and use `OperatingSystemMXBean` to check the number of open file descriptors. Its faster and more succinct.

Reply:
`lsof` is a perfectly fine command, I don't see any needs to use anything else.
 - The test is checking that there are no file descriptor leaks, I don't see why we need a unit test here. 
 - it uses docker containers which guaranties same result on different operation systems, which is not possible with a unit test (in case of lsof).
 - I believe lsof is more solid approach than anything else including JMX.

Reply:
Also if you are talking about [UnixOperatingSystemMXBean](https://docs.oracle.com/javase/8/docs/jre/api/management/extension/index.html?com/sun/management/UnixOperatingSystemMXBean.html )
unfortunately, it doesn't provide enough information regarding what files are open and their statuses.